src/Sentry.Maui/BindableSentryMauiOptions.cs

```diff
@@ -0,0 +1,19 @@
+using Sentry.Extensions.Logging;
+
+namespace Sentry.Maui;
+
+/// <inheritdoc cref="BindableSentryOptions"/>
+internal class BindableSentryMauiOptions : BindableSentryLoggingOptions
```

Comment:
I see. Because the `BindableMauiOptions` inherit from the `BindableLoggingOptions`. But why was it not necessary before?